### PR TITLE
(main) Release: 2.24.8

### PR DESCRIPTION
**Pull request by taylorjdawson**

### CODE REVIEW

**Comment by Adamj1232** [comment]
Looks like the `docs` and `main` branches need to be merged into the release branch

File: packages/injected/README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ console.log(connectedWallets)
 - BifrostWallet - _Desktop & Mobile_
 - Safeheron - _Desktop_
 - Talisman - _Desktop_
+- Coin98 Wallet - _Desktop & Mobile_
```
Comment (Adamj1232) [inline packages/injected/README.md:93]:
```suggestion
- Ronin Wallet - _Desktop & Mobile_
- Coin98 Wallet - _Desktop & Mobile_
- SubWallet - _Desktop & Mobile_
```

These were missing in the README but are in the docs site

Comment (taylorjdawson) [reply packages/injected/README.md:93]: I merged latest docs just now how's that?

File: docs/package.json [hunk 2]
```diff
@@ -49,15 +49,15 @@
...
+    "@web3-onboard/coinbase": "^2.2.5",
+    "@web3-onboard/core": "^2.21.0",
     "@web3-onboard/dcent": "^2.2.7",
     "@web3-onboard/enkrypt": "^2.0.4",
     "@web3-onboard/fortmatic": "^2.0.19",
-    "@web3-onboard/frame": "^2.0.2-alpha.1",
+    "@web3-onboard/frame": "^2.0.2",
     "@web3-onboard/frontier": "^2.0.4",
     "@web3-onboard/gas": "^2.1.8",
     "@web3-onboard/gnosis": "^2.1.10",
```
Comment (Adamj1232) [inline docs/package.json:63]: I think the new gnosis deps will need to be added to the docs and react demo to work

Comment (taylorjdawson) [reply docs/package.json:63]: would that be in the docs PR or the main pr here?

**Review by Adamj1232 — APPROVED** [review]

**Review by leightkt — APPROVED** [review]

**Pull request merged**